log-collector-script/linux/eks-log-collector.sh

```diff
@@ -385,6 +393,11 @@ get_k8s_info() {
         systemctl cat "${entry}" > "${COLLECT_DIR}"/kubelet/"${entry}"_service.txt 2>&1
       done
       ;;
+    snap)
+      timeout 75 snap logs kubelet-eks -n all > "${COLLECT_DIR}"/kubelet/kubelet.log
+
+      timeout 75 snap get kubelet-eks > "${COLLECT_DIR}"/kubelet/"${entry}"_service.txt 2>&1
```

Comment:
I think this line has been a bit mixed up compared to the the previous switch statement. I assume we want both `kubelet` and `kube-proxy` logs here as well, and the systemctl output? Or is `kube-proxy` started some other way?
```
      timeout 75 snap logs kubelet-eks -n all > "${COLLECT_DIR}"/kubelet/kubelet.log
      timeout 75 snap logs kube-proxy -n all > "${COLLECT_DIR}"/kubelet/kubeproxy.log

      for entry in kubelet-eks kube-proxy; do
        systemctl cat "${entry}" > "${COLLECT_DIR}"/kubelet/"${entry}"_service.txt 2>&1
      done
```

Reply:
kube-proxy is configured as daemon-set . So, there wouldn't be any logs from systemd or snap. i think we need to remove those from systemd as well. What you say ?
```
systemd)
      timeout 75 journalctl --unit=kubelet --since "${DAYS_10}" > "${COLLECT_DIR}"/kubelet/kubelet.log
      timeout 75 journalctl --unit=kubeproxy --since "${DAYS_10}" > "${COLLECT_DIR}"/kubelet/kubeproxy.log

      for entry in kubelet kube-proxy; do
        systemctl cat "${entry}" > "${COLLECT_DIR}"/kubelet/"${entry}"_service.txt 2>&1
      done
```

Reply:
Yes, that sounds right. Also that you can't use `"${entry}"` in the output file path for the `snap get` command.

Reply:
Update the kubelet service details for snap and systemd. Also, changed the way we collect the kubeconfig when kubectl is not available.